### PR TITLE
Bump node-manager to v0.1.1

### PR DESCRIPTION
**Pull request by chrisho**
add Ksmtuned merge_across_nodes

**Related issue**
https://github.com/harvester/harvester/issues/2827

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]
LGTM,can u squash the commits, thanks.

**Review by futuretea — APPROVED** [review]
LGTM

File: charts/harvester-vm-import-controller/templates/service.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "vm-import-controller.fullname" . }}
```
Comment (futuretea) [inline charts/harvester-vm-import-controller/templates/service.yaml:4]: not about node-manager.

**Review by futuretea — APPROVED** [review]
LGTM

**Pull request merged**